gno.land/pkg/gnoland/app.go

```diff
@@ -223,6 +224,9 @@ type InitChainerConfig struct {
 	// called several times.
 	CacheStdlibLoad bool
 
+	// ChainDomain is the primary domain name for the chain and its packages.
+	ChainDomain string
```

Comment:
why add this here instead of in genesis?

Reply:
I think that I prefer your idea. I'm waiting for more feedback before making the change.

Reply:
Yeah I agree it should be in genesis. Allows us more easily to find problems when there's a mismatch between two nodes.

Reply:
switching the PR to draft until I finish and merge #3003.

Reply:
done.